pages/room/[id].js

```diff
@@ -48,6 +48,26 @@ export default function Room(props) {
           )
         })}
       </div>
+      <div className="flex justify-center">
+        <button
```

Comment:
I agree with Shaun, I don't think the buttons should be `type=submit` since they are not in a form.